app/facades/header_facade.rb

```diff
@@ -0,0 +1,15 @@
+class HeaderFacade
```

Comment:
why is the class called HeaderFacade?

Reply:
I assume `app/facade` is the new "model" that will deal with the API service calls (rather than a postgres database model). Is that correct?